.travis.yml

```diff
@@ -54,5 +71,4 @@ deploy:
     skip_cleanup: true
     on:
       branch: master
-      python: 3.6
-      condition: '$BUILD_DOCS == "true"'
+      condition: '${PYTHON_VERSION} == 3.7 AND ${BUILD_DOCS} == "true"'
```

Comment:
I think this line is right... haven't seen it run, as it'll need a merge onto master, so I'm assuming it's okay...